Sources/SharedServices/DynamoUserRepository.swift

```diff
@@ -4,6 +4,16 @@ import Models
 import Extensions
 
 struct DynamoUserRepository {
+    enum Error: Swift.Error {
+        case discordUserNotFound(id: String)
+
+        var localizedDescription: String {
```

Comment:
Rename to `description`.

Realistically, i don't think you ever want to use `localizedDescription`.
What you usually want to use is a conformance to `CustomStringConvertible` followed by it's requirement, aka having `var description: String`. 
When you want a string representation of any type, Swift will have to resort to using `Mirror` to find that type's info at runtime, and generate a description for it (e.g. in string interpolation).
But when a type is conformed to `CustomStringConvertible`, Swift will be smart enough to use the `description` as the string representation, instead of using `Mirror`. This is good because using `Mirror` is known to be very slow.

Optionally, you might want to conform your Error types to `LocalizedError` too (need to import `Foundation`), and add a `var errorDescription: String?`. This is only useful for places that use Objective C runtime, like on macOS, iOS etc... . Overriding `errorDescription` will basically make it so you never see those unhelpful `The operation couldn’t be completed` errors, which is related to when an `Error` is converted to `NSError` and then you print it.